discover/analytics-cloud/articles/01-what-is-liferay-analytics-cloud.markdown

```diff
@@ -0,0 +1,124 @@
+# What is Liferay Analytics Cloud?
+
+Your company interacts with lots of people, including existing and prospective
+customers. On your websites they browse products and services, ask questions,
+leave feedback, etc. Presenting relevant, interesting website content motivates
+people to do business with you. Knowing your customers well and understanding
+how they interact with existing digital content can help you improve their
+experience. Liferay Analytics Cloud generates powerful, intuitive charts that
+deepen your knowledge of people and their online behavior so you can create
+terrific user experiences that drive business. 
+
+It all starts with understanding people.
+
+## Understanding People 
+
+Existing and prospective customers interact with your company in person, on your
+websites, and via email and messaging. On Liferay Digital Enterprise (DE)
+websites, people are registered users or visitors. Liferay Analytics Cloud
+(Analytics Cloud) integrates contacts from your Liferay DE instances and CSV
```

Comment:
Probably too much detail at this point. We don't want to mention technical details like CSV files till we get to documenting something. Consider changing to "....integrates contacts from multiple sources into a single view."